controllers/storagecluster/quickstart_constants.go

```diff
@@ -1,200 +1,181 @@
+    In this tour, you'll learn how to add block, file, or object storage to your applications, and how to monitor your storage resources in the OpenShift Data Foundation dashboards.  
 
- 1. Connecting applications to block or file storage (persistent volumes)
+      1. Connecting applications to block or file storage (persistent volumes)  
 
- 2. Connecting applications to object storage (object buckets)
+      1. Connecting applications to object storage (object buckets)  
 
- 3. Using the dashboards to monitor OpenShift Container Storage resources
+      1. Using the dashboards to monitor OpenShift Data Foundation resources
 
- "
   tasks:
     -
-      title: "Connecting applications to block or file storage (persistent volumes)"
-      description: "If you want yours data to live longer than yours pods, you need persistent volumes.
-
-
-   Persistent volumes exist outside the pod lifecycle, meaning that your data is retained even after your pod has been restarted, rescheduled, or deleted. You might use persistent volumes with a MySQL or WordPress application to store information for longer than the lifetime of any individual application pod.
-
-
-   After an administrator has set up an OpenShift Container Storage cluster, developers can use persistent volume claims (PVCs) to request persistent volume (PV) resources without needing to know anything specific about the underlying storage infrastructure.
+      title: Connecting applications to block or file storage (persistent volumes)
+      description: >-
 
+        If you want yours data to live longer than yours pods, you need persistent volumes.  
+      
+        Persistent volumes exist outside the pod lifecycle, meaning that your data is retained even after your pod has been restarted, rescheduled, or deleted. You might use persistent volumes with a MySQL or WordPress application to store information for longer than the lifetime of any individual application pod.  
 
-   **Connect your application with a PVC:**
 
+        After an administrator has set up an OpenShift Data Foundation cluster, developers can use persistent volume claims (PVCs) to request persistent volume (PV) resources without needing to know anything specific about the underlying storage infrastructure.
 
-   1. Click Workloads -> Deployments in the navigation menu on the left.
 
-   2. Select your project from the Project drop-down and find your application in the list of deployments.
-
-   3. Click the Action menu (⋮)-> Add Storage
-
-   4. Select \"Create new claim\". You would select \"Use existing claim\" here if you wanted to restore an existing persistent volume to a redeployed application.
+        **Connect your application with a PVC**
+ 
+           1. Click Workloads > Deployments in the navigation menu on the left.  
+   
+           2. Select your project from the Project drop-down and find your application in the list of deployments.  
+   
+           3. Click the Action menu &*⋮&* > Add Storage  
```

Comment:
Just a quick note on this line. 
I am replacing (&*) with ` (backtick)  as  go multiline strings do not support escaping backtick . The string enclosed by backticks are highlighted in markdown.

Reply:
If anyone got a better method for using backticks with multiline strings. they could suggest it here.